runner/ansible/roles/1.2.2/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Set Test ID
+  set_fact:
+    test_id: '1.2.2'
+
+- name: "{{ test_id }}.check"
+  shell: |
+   timeout=$(crm_attribute -t crm_config -G -n stonith-timeout --quiet)
+          if [[cibadmin -Q --xpath "//primitive[@type='fence_azure_arm']/@type" > /dev/null 2>&1 ]]; then
+            if [[ "${timeout}" = "900" ]]; then
```

Comment:
Here, in order of having `900`, this should go in the expected values map. `900` is the expected value for the scenario where we have the `fence_azure_arm`, and `144` for sbd.
We would need most probably 2 expected values, 1 for `fence_azure_arm` and other for `sbd`. Later on, you can just compare the `timeout` value. You don't need to do this `correct` outputting. This was done this way due a limitation of benchmark

Reply:
Will work this one out with you in chat.